Gemfile.lock

```diff
@@ -157,10 +157,10 @@ GEM
     msgpack (1.4.2)
     nanoid (2.0.0)
     nio4r (2.5.7)
-    nokogiri (1.11.1)
+    nokogiri (1.11.2)
       mini_portile2 (~> 2.5.0)
       racc (~> 1.4)
-    nokogiri (1.11.1-x86_64-darwin)
+    nokogiri (1.11.2-x86_64-darwin)
```

Comment:
@robbiepaul @mattonfoot @nomasprime Does anyone know if this use of a `darwin` variant will impact life on linux? I assume not as it currently runs with the previous version in CI which is a linux variant. But I have not before seen a gem tagged with `-darwin`

Reply:
Think if this was an issue, the CI wouldn't be green.